### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: 

Data correction for item IDs 526, 542, 543, 552, 564, 569, 601, 602, 610, 671, 675, 1132

https://www.divine-pride.net/database/item/526
https://www.divine-pride.net/database/item/542
https://www.divine-pride.net/database/item/543
https://www.divine-pride.net/database/item/552
https://www.divine-pride.net/database/item/564
https://www.divine-pride.net/database/item/569
https://www.divine-pride.net/database/item/601
https://www.divine-pride.net/database/item/602
https://www.divine-pride.net/database/item/610
https://www.divine-pride.net/database/item/671
https://www.divine-pride.net/database/item/675
https://www.divine-pride.net/database/item/1132

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -901,7 +901,7 @@ Body:
     EquipLevelMin: 40
     Refineable: true
     Script: |
-      bonus2 bAddEff,Eff_Curse,30;
+      bonus2 bAddEff,Eff_Curse,300;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:904]: 30 according to aegis data, do you know if/when it has been changed in the description?

File: db/re/item_db_usable.yml [hunk 2]
```diff
@@ -493,15 +492,15 @@ Body:
     Buy: 10
     Weight: 10
     Script: |
-      itemheal rand(325,405),0;
+      itemheal 365,0;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:495]: 'Restores around 365', please revert

File: db/re/item_db_usable.yml [hunk 3]
```diff
@@ -493,15 +492,15 @@ Body:
...
   - Id: 543
     AegisName: Festive_Rainbow_Cake
     Name: Festival Rainbow Cake
     Type: Healing
     Buy: 10
     Weight: 10
     Script: |
-      itemheal rand(325,405),0;
+      itemheal 365,0;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:503]: 'Restores around 365', please revert

File: db/re/item_db_usable.yml [hunk 4]
```diff
@@ -587,7 +586,7 @@ Body:
     Buy: 1
     Weight: 10
     Script: |
-      itemheal rand(70,90),rand(20,30);
+      itemheal rand(70,90),0;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:589]: It was fine according to aegis data and there is no value in kro description, please revert

File: db/re/item_db_usable.yml [hunk 5]
```diff
@@ -689,7 +688,7 @@ Body:
     Flags:
       BuyingStore: true
     Script: |
-      itemheal 200,0;
+      itemheal rand(175,234),0;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:691]: It was fine according to aegis data and there is no value in kro description, please revert

File: db/re/item_db_usable.yml [hunk 6]
```diff
@@ -742,7 +741,7 @@ Body:
     Flags:
       BuyingStore: true
     Script: |
-      itemheal rand(44,66),0;
+      itemheal rand(22,33),0;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:744]: It was fine according to aegis data and there is no value in kro description, please revert

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:904]: Capture

File: db/re/item_db_usable.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_usable.yml]: Thanks !

File: db/re/item_db_usable.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_usable.yml]: Thanks !

File: db/re/item_db_usable.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_db_usable.yml]: Thanks !

File: db/re/item_db_usable.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_db_usable.yml]: Thanks !

File: db/re/item_db_usable.yml (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_db_usable.yml]: Thanks !

**Comment by mazvi** [comment]
Add this item https://www.divine-pride.net/database/item/23475/infinity-drink

File: db/re/item_db_equip.yml [hunk 7]
```diff
@@ -1798,7 +1798,7 @@ Body:
       NoMail: true
       NoAuction: true
     Script: |
-      bonus bCritical,30;
+      bonus bCriticalRate,30;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: thanks

File: db/re/item_db_equip.yml [hunk 8]
```diff
@@ -1855,7 +1855,7 @@ Body:
     EquipLevelMin: 55
     Refineable: true
     Script: |
-      bonus bCritical,10;
+      bonus bCriticalRate,10;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: thanks

**Comment by idk-whoami** [comment]
> Add this item https://www.divine-pride.net/database/item/23475/infinity-drink

@mazvi  Do you have item script?

**Comment by mazvi** [comment]
@idk-whoami in here the script https://github.com/rathena/rathena/issues/5405

**Comment by Balferian** [comment]
id; 13295 - missed subtype
`SubType: Kunai`

**Comment by idk-whoami** [comment]
Thanks to @mazvi and @Balferian !

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**